### ISSUE TITLE
Create diagram for viewing/sun geometry angles in view extension

### ISSUE THREAD

**Issue by m-mohr** [stac-extensions/view#2]
@matthewhanson's all time favorite migrated from https://github.com/radiantearth/stac-spec/issues/653:

> Working on creating a diagram clarifying the angles used in the view extension (#644 )
> 
> * view_angle
> * incidence_angle
> * azimuth_angle
> * sun_azimuth_angle
> * sun_elevation_angle

**Comment by tschaub** [comment]
The [current text](https://github.com/stac-extensions/view/blob/e7bc7986a4ca2193e4c8370d727667b7021e1e62/README.md#item-properties) in the item properties table sounds like it describes these angles for `view:off_nadir` (A) and `view:incidence_angle` (B).

<img width="1018" alt="image" src="https://user-images.githubusercontent.com/41094/156231638-5e0faf8d-5fa8-4c3c-a543-60deb64eafe5.png">

> (A) `view:off_nadir`: The angle from the sensor between nadir (straight down) and the scene center. Measured in degrees (0-90).
> (B) `view:incidence_angle`: The incidence angle is the angle between the vertical (normal) to the intercepting surface and the line of sight back to the satellite at the scene center. Measured in degrees (0-90).

But then the narrative and example sound like they are talking about the 90 degree complement to this (or the grazing angle).

**Comment by remi-braun** [comment]
I agree with this picture, and I am very confused about the narrative, especially if I look to the litterature, such as [Airbus angle converter](https://www.intelligence-airbusds.com/en/8719-angle-conversion).
![](https://www.intelligence-airbusds.com/files/pmedia/public/r34890_9_illustration-conversion-angle_en.png)

For me it should be `incidence angle ~= off-nadir angle`, which is IMO more widely used.

**Comment by CloudNiner** [comment]
Hi! Could you clarify `azimuth` for us? Is it the left panel or the right panel in the associated image, or something else?

<img width="1718" alt="image" src="https://user-images.githubusercontent.com/1818302/174896641-0c47fc87-cc2f-4fe0-b6df-5bd004780097.png">

**Comment by matthewhanson** [comment]
The azimuth would be the angle measures from the sat to the scene center,
so your first diagram.

**Comment by matthewhanson** [comment]
@remi-braun I'm revisiting this extension while reviewing some updates to some stactools packages.

In your diagram, the Viewing Angle is what this extension calls `off_nadir`.  `incidence_angle` is then the complement (if a flat earth) of `off_nadir`.   

@tschaub is correct in what the angles are supposed to be, but I'm reviewing the text to see where it could be clarified.

**Comment by tschaub** [comment]
> I'm reviewing the text to see where it could be clarified.

This is the part that needs review:
https://github.com/stac-extensions/view/blob/914b4aebc8d45c1ff90308e57bbc2626f2e6727d/README.md#L37-L38

Instead of "the two angles sum to about 90," it could say that "the two angles are almost equal."

**Comment by remi-braun** [comment]
> @remi-braun I'm revisiting this extension while reviewing some updates to some stactools packages.
> 
> In your diagram, the Viewing Angle is what this extension calls `off_nadir`. `incidence_angle` is then the complement (if a flat earth) of `off_nadir`.
> 

Ok that's what I've done, thanks for the confirmation 😄

**Comment by tschaub** [comment]
> incidence_angle is then the complement (if a flat earth) of off_nadir

This is incorrect.  The incidence angle is not the complement of the off nadir angle.  In a flat earth model, they are equal.  In the real world, they are close to equal.

**Comment by matthewhanson** [comment]
Oh right, sorry @tschaub is correct....not sure why I wrote they are complements, in the flat earth model they are alternate interior angles which are equal. (just double checked the terminology with my wife who is a math teacher)
